debian/rules

```diff
@@ -9,8 +9,8 @@ override_dh_auto_configure:
 override_dh_auto_build:
 	ninja -C debian/build
 
-override_dh_auto_test:
-	ninja -C debian/build test
+#override_dh_auto_test:
```

Comment:
IMO the correct solution would be to wrap ninja invocation into ifeq like this:
```
override_dh_auto_test:
ifeq ($(filter nocheck,$(DEB_BUILD_OPTIONS)),)
        ninja -C debian/build test
endif
```

and then use `DEB_BUILD_OPTIONS=nocheck dpkg-buildpackage` when you want to skip the tests during the package build.